Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -267,10 +267,10 @@ void CSSParserImpl::parseStyleSheetForInspector(const String& string, const CSSP
 {
     CSSParserObserverWrapper wrapper(observer);
     CSSParserImpl parser(context, string, styleSheet, &wrapper);
-    bool firstRuleValid = parser.consumeRuleList(parser.tokenizer()->tokenRange(), TopLevelRuleList, [&styleSheet](RefPtr<StyleRuleBase> rule) {
+    bool firstRuleValid = parser.consumeRuleList(parser.tokenizer()->tokenRange(), TopLevelRuleList, [&styleSheet](Ref<StyleRuleBase> rule) {
```

Comment:
Again, could we be moving the rules here?

Reply:
Ditto